spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cloudfoundry/AzureCfService.java

```diff
@@ -9,45 +9,70 @@
 import com.google.common.collect.ImmutableMap;
 
 import java.util.Map;
+import java.util.stream.Collectors;
 
 enum AzureCfService {
 
     SERVICEBUS("servicebus", "azure-servicebus", ImmutableMap.of("connectionString", "connection-string")),
-    EVENTHUB("eventhub", "azure-eventhub", ImmutableMap.of("ConnectionString", "connection-string")),
-    STORAGE("storage", "azure-storage",
-            ImmutableMap.of("storageAccountName", "account", "accessKey", "access-key"));
+    EVENTHUB("eventhub", "azure-eventhub", ImmutableMap.of("connectionString", "connection-string")),
+    STORAGE("storage", "azure-storage", ImmutableMap.of("storageAccountName", "account", "accessKey", "access-key")),
+    REDIS("spring.redis", "azure-rediscache", ImmutableMap.of("host", "host", "password", "password", "port", "port")
```

Comment:
Why it's `spring.redis` not `azure.redis`?

Reply:
That's spring data redis standard property, not defined by spring cloud azure